conans/client/cache/cache.py

```diff
@@ -99,6 +100,19 @@ def installed_as_editable(self, ref):
     def config_install_file(self):
         return os.path.join(self.conan_folder, "config_install.json")
 
+    def remove_reference_system_reqs(self, ref):
```

Comment:
I think these functions don't belong to the cache but to the conan_api. I would say the command should check if it is a valid ref or not and then call to the right method in the `conan_api`. The names might be `remove_system_reqs_by_pattern(pattern)` or `remove_system_reqs(ref)`